server/command/command.go

```diff
@@ -73,21 +75,30 @@ func addSubCommands(splunk *model.AutocompleteData) {
 
 func createAlertCommand() *model.AutocompleteData {
 	alert := model.NewAutocompleteData(
-		"alert", "--subscribe", "subscribe to alert")
+		"alert", "[command]", "Available commands: subscribe, list, delete")
+
+	subscribe := model.NewAutocompleteData(
+		"subscribe", "", "Subscribe to an alert")
+		alert.AddCommand(subscribe)
```

Comment:
Lint is broken due to whitespace here

```suggestion
    alert.AddCommand(subscribe)
```